docs/post-cloner.md

```diff
@@ -0,0 +1,71 @@
+# Post Cloner
+
+This feature adds a button to the post edit screen that allows a user to clone a post with its metadata and terms to a new, identical post.
+
+It is enabled by default but can be disabled via the configuration file:
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"cms": {
+					"cloner": false
+				}
+			}
+		}
+	}
+}
+```
+
+## Actions
+
+**`post_cloner_after_meta_data_copy: (int) $post_id`**
+
+Fires when post meta has been copied to cloned post.
+
+## Filters
+
+**`post_cloner_cloned_parent: (int) $original_post['ID']`**
```

Comment:
Seems odd to show the variable that is actually passed to this filter - when using the filter you would name this parameter `$post_id` I think - it doesn't need to match the way it's called directly.